ddtrace/context.py

```diff
@@ -20,25 +20,31 @@ class Context(object):
 
     This data structure is thread-safe.
     """
-    def __init__(self, trace_id=None, span_id=None, sampled=True):
+    def __init__(self, trace_id=None, span_id=None, sampled=True, sampling_priority=None):
         """
         Initialize a new thread-safe ``Context``.
 
         :param int trace_id: trace_id of parent span
         :param int span_id: span_id of parent span
         """
         self._trace = []
-        self._sampled = sampled
         self._finished_spans = 0
         self._current_span = None
         self._lock = threading.Lock()
-        self._parent_span_id = span_id
+
         self._parent_trace_id = trace_id
+        self._parent_span_id = span_id
+        self._sampled = sampled
+        self._sampling_priority = sampling_priority
+
+    def get_context_attributes(self):
```

Comment:
I don't especially defend nor like this API.
What is sure, it is that we will need somewhere a public way to access the current tuple (tid, sid, priority) for remote propagation (both for our own integrations and for customers to instrument their own inter service propagation). Do you agree too?
After that, it could live anywhere.